amy/consents/forms.py

```diff
@@ -105,6 +105,19 @@ class RequiredConsentsForm(BaseTermConsentsForm):
     Builds form shown on login when there are missing required consents.
     """
 
+    def _build_form(self, person: Person) -> None:
```

Comment:
Is there a reason why `self.terms` cannot be passed to this method as an argument similarly to `person`?

Reply:
There's not. I'll change that.